themes/light.json

```diff
@@ -0,0 +1,16 @@
+{
+  "bgImagePath": "../images/background_light.svg",
+  "bgPrimary": "#fff",
+  "bgSecondary": "#f5f5f5",
+  "accentColor": "#2090ea",
+  "textPrimary": "#010101",
+  "textSecondary": "#62656a",
+  "ovalButtonBg": "#415e6b",
+  "ovalButtonText": "#fff",
+  "bgChatView": "lime",
```

Comment:
change this to `"bgChatView": "#e6dcd4"` I forgot to do that lime looks really ugly.